orcestra_book/reports/ATR-20240825a.md

```diff
@@ -0,0 +1,56 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+platform: ATR
+flight_id: ATR-20240825a
+takeoff: "2024-08-25 19:15:00Z"
+landing: "2024-08-25 22:45:00Z"
+departure_airport: GVAC
+arrival_airport: GVAC
+crew:
+  - name: 
+    job: Pilot
+  - name: 
+    job: Pilot
+  - name: 
+    job: Mechanics
+  - name: 
+    job: Expé Principal
+  - name: 
+    job: Expé 
+  - name: tbd
+    job: PI
+  - name: 
+    job: LNG
+  - name: 
+    job: aWALI
+  - name: 
+    job: Microphys 1
+  - name: 
+    job: Microphys 2
+  - name: 
+    job: RASTA
+  - name: 
+    job: BASTA
+categories: [sar_under, c_atr]
+orphan: true
+---
+
+# Flight Report - {front}`flight_id`
+
+```{badges}
+```
+
+## Report
+
+* **[Flight Report](https://thredds-x.ipsl.fr/thredds/fileServer/MAESTRO/INSITU/AIRCRAFT/ATR/REPORTS/ATR-20240825_RF12_as34_Flight_Report_MAESTRO.pdf)** from MAESTRO Operational Data Center
+
+* [Cached Flight Report](./REPORT-ATR-20240825a.pdf) as of 29 August 2024.
```

Comment:
Is there a need to duplicate the flight report, if we also link to an external source?

Reply:
Perhaps not anymore since the link appears to be stable (at least on the order of days). I'll go through the reports and delete the linked pdfs.

Reply:
Okay I've made these changes.

Reply:
It was more a general question, because here you added both at the same time. If we don't trust the original reports to be stable, we could of course archive them. I just think it should be a well informed decision ☺️